src/MLJInterface.jl

```diff
@@ -389,17 +393,284 @@ MLJModelInterface.metadata_model(
     path="LightGBM.MLJInterface.LGBMClassifier",
     input=MLJModelInterface.Table(MLJModelInterface.Continuous),
     target=AbstractVector{<:MLJModelInterface.Finite},
-    weights=true,
-    descr="Microsoft LightGBM FFI wrapper: Classifier",
+    weights=true
+    # descr="Microsoft LightGBM FFI wrapper: Classifier",
 )
 
 MLJModelInterface.metadata_model(
     LGBMRegressor; # end positional args
     path="LightGBM.MLJInterface.LGBMRegressor",
     input=MLJModelInterface.Table(MLJModelInterface.Continuous),
     target=AbstractVector{MLJModelInterface.Continuous},
-    weights=true,
-    descr="Microsoft LightGBM FFI wrapper: Regressor",
+    weights=true
+    # descr="Microsoft LightGBM FFI wrapper: Regressor",
 )
 
+"""
```

Comment:
I'd like to ask that you revert most of the changes above these lines back (except the descr ones, after explanations)

Reply:
Oh oops, did not realize i made changes above these lines 😅 Will change back so we have proper git blames :)

Reply:
I can't tell if this has been addressed. @josephsdavid Please mark resolved if addressed.